examples/wasm-jupyterlite/pixi.toml

```
@@ -0,0 +1,44 @@
+[project]
+name = "wasm"
+version = "0.1.0"
+description = "Setup a pixi environment for building a jupyterlite app with WASM"
+authors = ["Wolf Vollprecht <w.vollprecht@gmail.com>"]
+platforms = ["emscripten-wasm32", "osx-arm64", "linux-64", "win-64"]
```

```suggestion
platforms = ["emscripten-wasm32", "osx-arm64", "osx-64", "linux-64", "win-64"]
```
Or doesn't it work for that?